extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -67,14 +68,19 @@
     private YoutubeParsingHelper() {
     }
 
-    private static final String HARDCODED_CLIENT_VERSION = "2.20200214.04.00";
+    private static final String HARDCODED_CLIENT_VERSION = "2.20210429.07.00";
+    private static final String HARDCODED_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8";
```

Comment:
Would it make sense to add `AIzaSyA8eiZmM1FaDVjRy-df2KTyQ_vz_yYM39w` as well? (It's the one used on Android)

Reply:
I am not using the Android API right now in this PR but yes, if we use it in the Extractor, we should add it.